tst/EnergyPlus/unit/StandardRatings.unit.cc

```diff
@@ -339,6 +339,86 @@ TEST_F(EnergyPlusFixture, ChillerIPLVTest)
+    state->dataCurveManager->PerfCurve(CurveNum).InterpolationType = InterpTypeEnum::EvaluateCurveToLimits;
+    state->dataCurveManager->PerfCurve(CurveNum).Name = "ElectricEIRChiller McQuay WSC 471kW/5.89COP/Vanes EIRFT";
+    state->dataCurveManager->PerfCurve(CurveNum).Coeff1 = 4.475238E-01;
+    state->dataCurveManager->PerfCurve(CurveNum).Coeff2 = -2.588210E-02;
+    state->dataCurveManager->PerfCurve(CurveNum).Coeff3 = -1.459053E-03;
+    state->dataCurveManager->PerfCurve(CurveNum).Coeff4 = 4.342595E-02;
+    state->dataCurveManager->PerfCurve(CurveNum).Coeff5 = -1.000651E-03;
+    state->dataCurveManager->PerfCurve(CurveNum).Coeff6 = 1.920106E-03;
+    state->dataCurveManager->PerfCurve(CurveNum).Var1Min = 7.22;
+    state->dataCurveManager->PerfCurve(CurveNum).Var1Max = 12.78;
+    state->dataCurveManager->PerfCurve(CurveNum).Var2Min = 12.78;
+    state->dataCurveManager->PerfCurve(CurveNum).Var2Max = 26.67;
+    state->dataChillerElectricEIR->ElectricEIRChiller(1).ChillerEIRFTIndex = 2;
+
+    // EIR=f(PLR)
+    CurveNum = 3;
+    state->dataCurveManager->PerfCurve(CurveNum).CurveType = CurveTypeEnum::Cubic;
+    state->dataCurveManager->PerfCurve(CurveNum).NumDims = 1;
+    state->dataCurveManager->PerfCurve(CurveNum).ObjectType = "Curve:Quadratic";
+    state->dataCurveManager->PerfCurve(CurveNum).InterpolationType = InterpTypeEnum::EvaluateCurveToLimits;
+    state->dataCurveManager->PerfCurve(CurveNum).Name = "ElectricEIRChiller McQuay WSC 471kW/5.89COP/Vanes EIRFPLR";
+    state->dataCurveManager->PerfCurve(CurveNum).Coeff1 = 2.778889E-01;
+    state->dataCurveManager->PerfCurve(CurveNum).Coeff2 = 2.338363E-01;
+    state->dataCurveManager->PerfCurve(CurveNum).Coeff3 = 4.883748E-01;
+    state->dataCurveManager->PerfCurve(CurveNum).Var1Min = 0;
+    state->dataCurveManager->PerfCurve(CurveNum).Var1Max = 1.15;
+    state->dataChillerElectricEIR->ElectricEIRChiller(1).ChillerEIRFPLRIndex = 3;
+
+    CalcChillerIPLV(*state,
+                    state->dataChillerElectricEIR->ElectricEIRChiller(1).Name,
+                    TypeOf_Chiller_ElectricEIR,
+                    state->dataChillerElectricEIR->ElectricEIRChiller(1).RefCap,
+                    state->dataChillerElectricEIR->ElectricEIRChiller(1).RefCOP,
+                    state->dataChillerElectricEIR->ElectricEIRChiller(1).CondenserType,
+                    state->dataChillerElectricEIR->ElectricEIRChiller(1).ChillerCapFTIndex,
+                    state->dataChillerElectricEIR->ElectricEIRChiller(1).ChillerEIRFTIndex,
+                    state->dataChillerElectricEIR->ElectricEIRChiller(1).ChillerEIRFPLRIndex,
+                    state->dataChillerElectricEIR->ElectricEIRChiller(1).MinUnloadRat,
+                    IPLV,
+                    Optional<const Real64>(),
+                    ObjexxFCL::Optional_int_const(),
+                    Optional<const Real64>());
+
+    EXPECT_DOUBLE_EQ(round(IPLV * 100) / 100, 5.54); // 18.56 IPLV
```

Comment:
New unit test using performance curves for the chiller dataset which returns a capacity modifier of ~0.82 at AHRI rated conditions.

Reply:
Stepped through the unit test code, confirmed that the ASHR % load are calculated corrected and the unit test demonstrates the code change well.